### PR TITLE
Create slap.ts

### PR DESCRIPTION
**Pull request by cornl1**
Add $SLAP token metrics

### CODE REVIEW

File: src/tokens/slap.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+const fetcher: SupplyFetcher = async (options = defaultFetcherOptions) => {
+  const blockFrost = getBlockFrostInstance(options);
+  const total = 100_000_000_000;
+  const treasuryRaw = await getAmountInAddresses(blockFrost, SLAP, [
+    "stake1ux366xgzh78hh8hk0ep0824pz7ha306uese7u644h7wn86quca6l3", // $slapventures
+  ]);
+
+  const burnRaw = await getAmountInAddresses(blockFrost, SLAP, [
+    "addr1w8qmxkacjdffxah0l3qg8hq2pmvs58q8lcy42zy9kda2ylc6dy5r4", //$burnsnek
```
Comment (longngn) [inline src/tokens/slap.ts:14]: what? this address doesn't have SLAP token

Comment (cornl1) [reply src/tokens/slap.ts:14]: It does have it :-)  https://cardanoscan.io/tokenHoldings/addr1w8qmxkacjdffxah0l3qg8hq2pmvs58q8lcy42zy9kda2ylc6dy5r4
Zrzut ekranu 2024-01-01 215340

Comment (cornl1) [reply src/tokens/slap.ts:14]: but if you check on Cardanoscan directly it actually doesn't find SLAP until you "expand" tokens list. Maybe it's a Cardanoscan error due to too many tokens on the wallet? No idea. But the tokens are 100% there - this is the transaction from 18th Dec: https://cardanoscan.io/transaction/667394df5e528e8f8268363a1580aff4ae06f1b6efee7fafec460603990973b7

**Comment by cornl1** [comment]
Checking if it's fine now - we are looking forward to having that info live :)

**Comment by cornl1** [comment]
hello sirs, how are we looking now?

**Review by shadowkora — APPROVED** [review]

**Pull request merged**